homeassistant/components/tibber/sensor.py

```diff
@@ -281,9 +243,14 @@ def __init__(self, tibber_home, sensor_name, device_class, unit, initial_state):
         """Initialize the sensor."""
         super().__init__(tibber_home)
         self._sensor_name = sensor_name
-        self._device_class = device_class
-        self._unit = unit
-        self._state = initial_state
+        self._model = "Tibber Pulse"
+
+        self._attr_device_class = device_class
+        self._attr_name = f"{self._sensor_name} {self._home_name}"
+        self._attr_should_poll = False
```

Comment:
As this is static, you might want to consider setting it like so:

```py
class TibberSensor():
    _attr_should_poll = False
    
    def __init__(...):
      ...
```

Keeps the constructor as small as possible.